### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by b4ldr**
Release 3.2.1

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
@alexjfisher suggested this before, but please use `Release 3.2.1` as the commit message.

**Comment by b4ldr** [comment]
updated

**Review by ekohl — COMMENTED** [review]
Neither change shows up in the resulting module on the forge (since we exclude those files) so I wonder what the point of releasing this is. For comparison, I built both using `rake module:build` (where I was lazy and used git master):
```console
$ diff -Nutr puppet-augeasproviders_core-3.2.0 puppet-augeasproviders_core-3.2.1-rc0
diff -Nutr puppet-augeasproviders_core-3.2.0/metadata.json puppet-augeasproviders_core-3.2.1-rc0/metadata.json
--- puppet-augeasproviders_core-3.2.0/metadata.json	2023-01-03 14:25:35.000000000 +0100
+++ puppet-augeasproviders_core-3.2.1-rc0/metadata.json	2023-01-03 14:24:01.000000000 +0100
@@ -1,6 +1,6 @@
 {
   "name": "puppet-augeasproviders_core",
-  "version": "3.2.0",
+  "version": "3.2.1-rc0",
   "author": "Vox Pupuli",
   "summary": "Alternative Augeas-based providers for Puppet",
   "license": "Apache-2.0",
```

**Comment by b4ldr** [comment]
@ekohl thanks, the main reason was for #43 but will ping the author of that to get more info

@lollipopman in relation to #43 how did you hit this issue, did you pull in this module from forge or git?  if the later where you using a specific tag or just master?

**Comment by lollipopman** [comment]
@b4ldr & @ekohl I used g10k to pull the modules directly from github, based on the v3.2.0 tag, which is how I ran into this issue.

**Comment by b4ldr** [comment]
just noting also that the puppet-posfix library [imports rspec files from the spec dir in this repo](https://github.com/bodgit/puppet-postfix/blob/main/spec/spec_helper_local.rb#L3). This is probably not the best idea but its the reason they are not using the forge module

**Comment by lollipopman** [comment]
> just noting also that the puppet-posfix library [imports rspec files from the spec dir in this repo](https://github.com/bodgit/puppet-postfix/blob/main/spec/spec_helper_local.rb#L3). This is probably not the best idea but its the reason they are not using the forge module

thanks @b4ldr, here is what the error looks like in CI, https://github.com/lollipopman/puppet-postfix/actions/runs/3832180811/jobs/6522193480#step:5:4355

**Comment by ekohl** [comment]
That looks like it doesn't include rspec-mocks. Perhaps it's because of https://github.com/lollipopman/puppet-postfix/blob/6fa18a696d86a6014889fd8c04e78a4b4cfb0dfd/spec/spec_helper.rb#L3-L5 which AFAIK disables `rspec-mocks`. I'd suggest to migrate the module to use rspec-mocks.

**Comment by ekohl** [comment]
Oh, I should also mention that it's because of https://github.com/voxpupuli/puppet-augeasproviders_core/commit/637108ebcdab86b2de29ff51b41dfc6db39ab4d9 where it was changed.

**Comment by lollipopman** [comment]
> Oh, I should also mention that it's because of [637108e](https://github.com/voxpupuli/puppet-augeasproviders_core/commit/637108ebcdab86b2de29ff51b41dfc6db39ab4d9) where it was changed.

Thanks @ekohl for looking at this, that commit is indeed what broke bodgit/puppet-postfix's test, as I mentioned in my pull request, https://github.com/voxpupuli/puppet-augeasproviders_core/commit/aa427ad6ea03dbec36c95c8d92c668509b2fb1cb

**Comment by bastelfreak** [comment]
@b4ldr can you please rebase after https://github.com/voxpupuli/puppet-augeasproviders_core/pull/44 was merged?

**Comment by h-haaks** [comment]
@b4ldr can you please rebase after #49 was merged?
And change your commit message to `Release 3.2.1`

**Comment by b4ldr** [comment]
> @alexjfisher suggested this before, but please use `Release 3.2.1` as the commit message.

done and rebased

**Review by ekohl — CHANGES_REQUESTED** [review]
Can you regenerate the changelog too? The date is off and by now we've merged a real change (workaround for augeas 1.13.0 bug).

**Comment by b4ldr** [comment]
> Can you regenerate the changelog too? The date is off and by now we've merged a real change (workaround for augeas 1.13.0 bug).

updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.2.1](https://github.com/voxpupuli/puppet-augeasproviders_core/tree/v3.2.1) (2023-04-26)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-augeasproviders_core/compare/v3.2.0...v3.2.1)
+
+**Merged pull requests:**
+
+- Workaround libaugeas save/load issue [\#49](https://github.com/voxpupuli/puppet-augeasproviders_core/pull/49) ([h-haaks](https://github.com/h-haaks))
```
Comment (ekohl) [inline CHANGELOG.md:13]: I've labeled this as abugfix

**Review by ekohl — APPROVED** [review]
If you regen the changelog one more time that PR should have the correct label. Feel free to release it then.

File: CHANGELOG.md (hunk 1 above)
Comment (b4ldr) [reply CHANGELOG.md:13]: done

**Comment by b4ldr** [comment]
updated

**Pull request merged**